cmd/truss/main.go

```diff
@@ -94,6 +94,7 @@ func main() {
 // service definition files.
 func parseInput() (*truss.Config, error) {
 	var cfg truss.Config
+	wd, err := os.Getwd()
```

Comment:
While the `err` variable is declared here, if `Getwd` actually did return an error the program would never know since `err` isn't checked and is later overwritten in another function call.